HHS/Head-Start-TTADP
# [TTAHUB-1521] - Display regions in AR Landing header/SiteNav

## Description of change
User's regions are displayed below their name in the SiteNav. If user has access to multiple regions, the AR landing page title will be "Activity reports - your regions" and if they only have access to one region, the title will be "Activity reports - your region". In site nav, there are 3 versions under the user's name: Region #, Regions #, #, #, and All Regions (see ticket with mockups). Also corrected the alignment of the "New Activity Report" button, and spacing between that button and filter button at smaller device widths.


## How to test
- Change user permissions to set user's home ID to region 14, with read permissions to all regions. Should see "All Regions" in SiteNav and "- your regions" in page title.
- Change user permissions to set user's home ID to region 1, with read permissions to region 1 and 5. Should see "Regions 1,5" in SiteNav and "- your regions" in page title.
- Change user permissions to set user's home ID to region 5, with read permissions to region 5. Should see "Region 5" in SiteNav and "- your region" in page title.
- Reduce window size to where "New Activity Report" button moves below page title. There should be 24px between title and "New Activity Report" button and 16px between Report button and filter button. 

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1521


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
